.travis.yml

```diff
@@ -3,13 +3,15 @@ node_js:
   - "0.11"
   - "0.10"
   - "0.8"
+  - "iojs"
+  - "iojs-v2.0.0"
+  - "iojs-v1.8.1"
+  - "iojs-v1.0.4"
```

Comment:
Since iojs is getting merged back into node.js, let's test only the latest flavor. Can we condense these to:

```
- "iojs"
```